packages/closer/utils/locale.helpers.ts

```
@@ -42,6 +42,14 @@ export async function loadLocaleData(
               import('../locales/base-en.json'),
             ]);
         }
+      case 'per-auset':
+        switch (locale) {
+          case 'en':
+            return Promise.all([
+              import('../locales/per-auset/en.json'),
+              import('../locales/base-en.json'),
+            ]);
+        }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)